juriscraper/pacer/email.py

```diff
@@ -45,3 +52,71 @@ def _get_date_filed(self):
         return convert_date_string(
             date_filed[0].lower().replace("filed on ", "")
         )
+
+    def _get_document_number(self):
+        document_number = self._get_document_number_from_link()
+        return (
+            document_number
+            if document_number is not None
+            else self._get_document_number_from_text()
+        )
+
+    def _get_document_number_from_link(self):
+        try:
+            path = '//td[contains(., "Document Number:")]/following-sibling::td[1]/a/text()'
+            return numbers_only(self.tree.xpath(path)[0])[0]
+        except IndexError:
+            return None
+
+    def _get_document_number_from_text(self):
+        try:
+            path = '//td[contains(., "Document Number:")]/following-sibling::td[1]/text()'
+            return numbers_only(self.tree.xpath(path)[0])[0]
+        except IndexError:
+            return None
```

Comment:
Could these be combined with:

```
path = '//td[contains(., "Document Number:")]/following-sibling::td[1]//text()'
```

Reply:
Does not appear to be the case, unfortunately. I did not love this solution, but was trying not to get bogged down in xpath land.